@navikt/core/css/form/combobox.css

```diff
@@ -120,6 +120,14 @@
   padding-left: 0.25rem;
 }
 
+.navds-combobox__selected-options[data-type="single"] {
+  gap: 0;
+}
+
+.navds-combobox__selected-options > li:not(:only-child) > .navds-combobox__input {
+  margin-left: 0;
+}
```

Comment:
We could probably do the same thing here (add the margin when we want it, instead of removing it when we don't want it)

Reply:
Done ✅